### PR TITLE
Added XPath scene scraper for EroProfile

### PR DESCRIPTION
**Pull request by imagineimaginingthings**
This will handle both the desktop and mobile websites despite their completely different HTML.

Here's a randomly selected video URL (obviously very NSFW) you can test it with both desktop and mobile versions:
```
https://www.eroprofile.com/m/videos/view/ggsdgbZNXB
https://m.eroprofile.com/p/videos/view/ggsdgbZNXB
```

### CODE REVIEW

File: scrapers/EroProfile.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      Image: //video[@id="vidVideoViewPlayer"]/@poster | //video[@id="vidPlayer"]/@poster
+      Date: 
+        selector: //div[@class="col-xs-3 col-sm-2 colEmp"][contains(text(),"Upload date")]/following-sibling::div/text() | //table[@class="data marT"]//tbody//tr//th[contains(text(),"Uploaded:")]/following::td/text()
+        postProcess:
+          - replace:
+              - regex: (.+)\s\((.+)ago\)
+                with: $1
+          - parseDate: 2 January 2006
+      Details: //h1[@class="capMultiLine"]/following::p/text() | //table[@class="data marT"]//tbody//tr//th[contains(text(),"Description:")]/following::td/text()
+
```
Comment (DogmaDragon) [inline scrapers/EroProfile.yml:21]: Unneccesary. Empty line.

**Comment by DogmaDragon** [comment]
Tested:
- Scenes work

**Review by JackDawson94 — COMMENTED** [review]
Tested - works as intended

**Review by JackDawson94 — APPROVED** [review]

**Comment by imagineimaginingthings** [comment]
The offending unnecessary empty line has now been removed

**Pull request merged**